### PR TITLE
#483 Copy images folder + fix missing assets

### PR DESCRIPTION
**Pull request by oubchid**

### CODE REVIEW

File: webpack.commons.js [hunk 1]
```diff
@@ -78,15 +78,7 @@ module.exports = {
...
-          from: path.resolve(__dirname, 'src', 'linagora.esn.unifiedinbox', 'images', 'white-logo.svg'),
-          to: 'images'
-        },
-        {
-          from: path.resolve(__dirname, 'src', 'linagora.esn.unifiedinbox', 'images', 'user.png'),
-          to: 'images'
-        },
-        {
-          from: path.resolve(__dirname, 'src', 'linagora.esn.unifiedinbox', 'images', 'logo-tiny.png'),
+          from: path.resolve(__dirname, 'src', 'linagora.esn.unifiedinbox', 'images'),
```
Comment (rezk2ll) [inline webpack.commons.js:81]: does this work? last time i checked I didn't copy the images folder for me

Comment (oubchid) [reply webpack.commons.js:81]: Yes It works, for me in dev mode after `npm run serve` I can see all the images copied !

Comment (rezk2ll) [reply webpack.commons.js:81]: alright, try compiling it in prod mode `npm run build:prod` and check the dist folder for the images. if everything is good we merge

Comment (oubchid) [reply webpack.commons.js:81]: > does this work? last time i checked I didn't copy the images folder for me

FYI; in dev mode you can add `writeToDisk` prop in your dev server (webpack.common.js) and images are copied to `/dist/images`

**Review by rezk2ll — APPROVED** [review]

File: webpack.commons.js (hunk 1 above)
Comment (oubchid) [reply webpack.commons.js:81]: > alright, try compiling it in prod mode `npm run build:prod` and check the dist folder for the images. if everything is good we merge

Works, in prod mode, files are generated in `dist/assets/images`

**Review by renaudboyer — APPROVED** [review]

**Pull request merged**